src/collections/programs/mlh-2020/index.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: "MLH Fellowship Program"
+thumbnail: ./mlh.svg
+link: "mlh/2020"
+published: true
+---
+
+import { ProgramsWrapper } from "../Programs.style.js";
+import Button from "../../../reusecore/Button";
+import mlh_logo from "./mlh.svg";
+
+<ProgramsWrapper>
+
+<img src={mlh_logo}/>
+<h2><b>MLH Fellowship Program</b></h2>
+<div>
+The Major League Hacking’s <a href="https://https//fellowship.mlh.io/">MLH Fellowship</a> is an internship alternative for software engineers. Instead of working on a project for just one company, you’ll contribute to Open Source projects that are used by companies around the world. You’ll collaborate with a group of 10 students under the guidance of a professional mentor whose only job is to help you successfully contribute.<br/>
+
+<br/>
+Layer5’s <a href="https://https//meshery.io/">Meshery Project</a> is participating in the MLH Fellowship’s internship program in 2020.
```

Comment:
Fix this link please

Reply:
wow, my bad. I missed that, would fix it asap

Reply:
Done!